### ISSUE TITLE
Manila: Staff Profile page: Left nav Leadership link points to incorrect path

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#19992]
## Description

On Manila, from a Staff profile page, the Leadership list left sidenav link has the wrong URL: 

Tugboat set up with test content: https://tugboat.vfs.va.gov/6749f5a1e2b213c935a1c93b

**Drupal:** Leadership list URLs are correct:
* Tugboat node: https://cms-z93u4oqvyg73spgvezmh9h94sr6jz4ps.demo.cms.va.gov/manila-va-clinic/about-us/leadership
* Prod: https://prod.cms.va.gov/manila-va-clinic/about-us/leadership

**FE**
* Tugboat correct Leadership list URL: https://web-z93u4oqvyg73spgvezmh9h94sr6jz4ps.demo.cms.va.gov/manila-va-clinic/about-us/leadership/ - correct URL
* From the [Staff profile page](https://web-z93u4oqvyg73spgvezmh9h94sr6jz4ps.demo.cms.va.gov/manila-va-clinic/about-us/leadership/) click Leadership in the left nav: https://web-z93u4oqvyg73spgvezmh9h94sr6jz4ps.demo.cms.va.gov/manila-va-system/about-us/leadership -- broken link

### To repro
* In a Tugboat env with a published Staff Profile page in the VA Manila system
* Visit the [Staff profile page](https://web-z93u4oqvyg73spgvezmh9h94sr6jz4ps.demo.cms.va.gov/manila-va-clinic/about-us/leadership/)
* Click Leadership in th eleft sidenav
* Land on a 404 page, with `/manila-va-system/about-us/leadership` in the path

## Acceptance criteria
- [ ] Leadership link in left sidenav should correctly point to the live / published Leadership list for Manila

**Comment by jilladams** [comment]
@Agile6MSkinner @eselkin I am pretty sure this new Manila issue is a FE fix required. But menus are weird, and we may find that it actually needs to get resolved in Drupal or is related to #19762, not sure.

**Comment by eselkin** [comment]
Probably a drupal child issue. I'll check.

**Comment by jilladams** [comment]
(Eli, sorry should have clarified -- We still have this editor in a holding pattern on making this type of content, so it's not urgent to resolve now, just flagging for awareness. )

I also am not seeing the new Staff Profile that I published get added to the Manila Leadership page: 
* Profile: https://web-z93u4oqvyg73spgvezmh9h94sr6jz4ps.demo.cms.va.gov/manila-va-clinic/staff-profiles/first-lastname/ (First Lastname)
* Leadership page: https://web-z93u4oqvyg73spgvezmh9h94sr6jz4ps.demo.cms.va.gov/manila-va-clinic/about-us/leadership/ - my new Staff person, First Lastname, doesn't appear. 

Not sure if that should be its own ticket or not.

**Comment by jilladams** [comment]
This happened for Lovell too, per Dave: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12943

**Comment by SnowboardTechie** [comment]
Was unblocked on this Tuesday and was able to get a working solution on a tugboat. Will be refining the PR for this today and moving onto #19406

**Comment by jilladams** [comment]
@SnowboardTechie one thing we don't have in ACs here that I will add now, is that we need to be mindful of Lovell, and make sure we don't break any logic for those links with the new change. 

e.g. 
* https://www.va.gov/lovell-federal-health-care-va/staff-profiles/robert-buckley/  
  * As a VA beneficiary, left sidenav Leadership link points to https://www.va.gov/lovell-federal-health-care-va/about-us/leadership
  * As a Tricare beneficiary, left sidenav link points to https://www.va.gov/lovell-federal-health-care-tricare/about-us/leadership

**Comment by SnowboardTechie** [comment]
Nice catch, ty @jilladams I went ahead and checked those out on the tugboat below and looks like we're all good, I see VA links when viewing as VA and Tricare when viewing as Tricare. The logic is built in a way that _**should**_ be fairly isolated to the manila namespace, but never hurts to be safe 😄 

- https://web-8zpd9j4sobjcc5dutp67lq4xmmmsj3x2.demo.cms.va.gov/lovell-federal-health-care-va/staff-profiles/robert-buckley/

**Comment by jilladams** [comment]
Built a tugboat with latest state of main for everything, and tested on a Staff Profile that has it's own page. Leadership left nav link correctly goes aback to the VA Manila leadership page. Closing!
